Docs/Configuration.tex

```diff
@@ -2998,66 +3018,68 @@ \subsection{Boot Properties}\label{miscbootprops}
   \end{itemize}
 
-  To see all entries picker menu needs to be reloaded in extended mode by pressing
-  \texttt{Spacebar} key. Hiding auxiliary entries may increase boot performance
-  for multidisk systems.
-
+  To display all entries, the picker menu can be reloaded into ``Extended Mode'' by pressing the
+  \texttt{Spacebar} key. Hiding auxiliary entries may increase boot performance on multidisk systems.
 
 \item
   \texttt{LauncherOption}\\
   \textbf{Type}: \texttt{plist\ string}\\
   \textbf{Failsafe}: \texttt{Disabled}\\
-  \textbf{Description}: Register the launcher option in firmware preferences for persistence.
+  \textbf{Description}: Register the launcher option in the firmware preferences for persistence.
 
   Valid values:
 
   \begin{itemize}
   \tightlist
   \item \texttt{Disabled} --- do nothing.
-  \item \texttt{Full} --- create or update top-priority
-  boot option in UEFI variable storage at bootloader startup. For this option
-  to work \texttt{RequestBootVarRouting} is required to be enabled.
+  \item \texttt{Full} --- create or update the top priority
+  boot option in UEFI variable storage at bootloader startup.
+  \begin{itemize}
+  \tightlist
+    \item Enable \texttt{RequestBootVarRouting} for this option to work.
+  \end{itemize}
   \item \texttt{Short} --- create a short boot option instead of a complete one.
-  This variant is useful for some older types of firmware, typically from Insyde,
-  that are unable to manage full device paths.
+  \begin{itemize}
+  \tightlist
+    \item This variant is useful for some older types of firmware, typically from Insyde,
+    that are unable to manage full device paths.
+  \end{itemize} \medskip
   \end{itemize}
 
-  This option provides integration with third-party operating system installation and upgrade
-  at the times they overwrite
-  \texttt{\textbackslash EFI\textbackslash BOOT\textbackslash BOOTx64.efi}
-  file. By creating a custom option in this file path becomes no longer
-  used for bootstrapping OpenCore. The path used for bootstrapping is specified
-  in \texttt{LauncherPath} option.
-
-  \emph{Note 1}: Some types of firmware may have faulty NVRAM, no boot option support, or other
-  incompatibilities. While unlikely, the use of this option may even cause boot failures.
-  This option should be used without any warranty exclusively on the boards known to be compatible.
-  Check \href{https://github.com/acidanthera/bugtracker/issues/1222}{acidanthera/bugtracker\#1222}
+  This option allows integration with third-party operating system installation and upgrades
+  (which may overwrite the \texttt{\textbackslash EFI\textbackslash BOOT\textbackslash BOOTx64.efi}
+  file). The BOOTx64.efi file is no longer used for bootstrapping OpenCore if a custom option is created.
+  The custom path used for bootstrapping can be specified by using the \texttt{LauncherPath} option.
+
+  \emph{Note 1}: Some types of firmware may have defective NVRAM implementation, no boot option
```

Comment:
I think that there should be a transition word. `hence` would make sense to me.

Reply:
No, this is correct, the NVRAM may work but there could be no boot option support.